temporary-plugins/hold-potentially-deprecate/superset-ui-legacy-plugin-chart-word-cloud/package.json

```diff
@@ -35,6 +35,6 @@
   "peerDependencies": {
     "@superset-ui/chart": "^0.14.0",
     "@superset-ui/color": "^0.14.0",
-    "@superset-ui/translation": "^0.14.0"
+    "@superset-ui/core": "^0.15.0"
```

Comment:
Is the intention to release 0.15 once this is merged?

Reply:
yes